src/Compilers/Core/Portable/SymbolDisplay/SymbolDisplayPartKind.cs

```diff
@@ -80,6 +80,8 @@ public enum SymbolDisplayPartKind
         ExtensionMethodName = 29,
         /// <summary>The name of a field or local constant.</summary>
         ConstantName = 30,
+        /// <summary>The name of a record.</summary>
+        RecordName = 31,
```

Comment:
Need to update line 89 & 98 to refrence RecordName instead of ConstantName.

Reply:
Thanks for the catch! Updated. I appreciate the helpful post above as well.